src/App.tsx

```diff
@@ -1,10 +1,67 @@
-import React from 'react';
+import React, {useState} from 'react';
+import axios from 'axios';
 import './App.css';
+import UserProfile from './UserProfile';
 
+
+const BASE_URL:string = "https://api.github.com/users/"; 
+let timeout:any;
 function App() {
+  const [userName, setUserName] = useState<string>("");
+  const [userData, setUserData] = useState<object>();
```

Comment:
Don't use `object`, You must Add a type that you are expecting, avoid using generic types which aren't give you details